libs/tools/shared/src/interfaces/package-json.ts

```diff
@@ -16,10 +16,15 @@
 
 export interface PackageJson {
   version?: string;
-  peerDependencies?: {
+  peerDependencies: {
```

Comment:
not all packageJson files have a peerDependency field
same for dev or normal dependencies - those fields should be optional

Reply:
Good point just copied it.